### PR TITLE
Update copy: offerings

### PR DESCRIPTION
**Pull request by timcasasola**
Updated copy to speak directly to client needs ("You want X. We'll do Y.").

### CODE REVIEW

File: views/elements/content/right.md [hunk 1]
```diff
@@ -4,10 +4,10 @@
...
+You have a vision for a new product. We'll bring it into clear focus, then collaboratively design a seamless product.
 
 ## Iterate on an existing product
-We’ll unpack opportunities to improve your product, and work with you to prioritize and design them.
+You want to improve your current product. We’ll unpack the opportunity, shape it into a tangible outcome, then design it.
 
 ## Augment your team
-We'll embed ourselves within your company to solve tough problems, and move your product forward, alongside your entire team.
\ No newline at end of file
+You need an interim product design team. We'll embed ourselves within your company to solve tough problems.
```
Comment (devinhalladay) [inline views/elements/content/right.md]: I'd say "interim" makes us sound like middlemen…can we change this to something like:

You need an in-house product design team, or to add new minds to your existing team. We'll embed ourselves within your company to solve tough problems.

Not previous about that copy, but I think making it clear that we can either augment your existing team or work with you as your in-house product team is important

**Review by devinhalladay — CHANGES_REQUESTED** [review]
@timcasasola just requested one change on the team augmentation section to make it sound more powerful and add that we can 1) augment an existing team or 2) be your first design team

File: views/elements/content/right.md (hunk 1 above)
Comment (timcasasola) [reply views/elements/content/right.md]: Word, sounds good to me!

**Comment by timcasasola** [comment]
@devinhalladay copy changes made per your feedback! Up to merge?

**Review by devinhalladay — APPROVED** [review]
lgtm, great work

**Pull request merged**